components/constellation/constellation.rs

```diff
@@ -872,6 +879,13 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             FromCompositorMsg::LogEntry(top_level_frame_id, thread_name, entry) => {
                 self.handle_log_entry(top_level_frame_id, thread_name, entry);
             }
+            FromCompositorMsg::SetWebVRThread(webvr_thread) => {
+                self.webvr_thread = Some(webvr_thread)
```

Comment:
Probably is worth sanity-checking with something like: `assert!(self.webvr_thread.is_none())`